domain/src/test/java/org/oppia/domain/exploration/ExplorationProgressControllerTest.kt

```diff
@@ -100,6 +102,12 @@ class ExplorationProgressControllerTest {
   @Mock
   lateinit var mockAsyncAnswerOutcomeObserver: Observer<AsyncResult<AnswerOutcome>>
 
+  @Mock
+  lateinit var mockAsyncHintObserver: Observer<AsyncResult<Hint>>
+
+  @Mock
+  lateinit var mockAsyncSolutionObserver: Observer<AsyncResult<Solution>>
```

Comment:
Optional Nit: These can be written in single lines.
`@Mock lateinit var mockAsyncSolutionObserver: Observer<AsyncResult<Solution>>`

Ditto everywhere else.

Reply:
Updated everywhere.